apps/wfo-ui/pages/index.tsx

```diff
@@ -11,10 +12,11 @@ import {
 
 export function Index() {
     const { data: session } = useSession();
+    const t = useTranslations(`main`);
```

Comment:
Is that a backtick or a single quote? Single quotes should be used here

Reply:
check